src/style.scss

```diff
@@ -144,35 +133,34 @@
   width: 52px;
   height: 12px;
   border-radius: 1.8rem;
-  background-color: var(--ds-advanced-color-button-primary-background, $ds-advanced-color-button-primary-background);
+  background-color: var(--ds-advanced-color-button-primary-background, v.$ds-advanced-color-button-primary-background);
 
   &::before {
     width: 64px;
   }
 }
 
 .embla__progress {
+  width: 52px;
+  height: 12px;
   border-radius: 1.8rem;
   border: none;
-  background-color: var(--ds-advanced-color-button-primary-background-inactive, $ds-advanced-color-button-primary-background-inactive);
+  background-color: var(--ds-advanced-color-button-primary-background-inactive, v.$ds-advanced-color-button-primary-background-inactive);
```

Comment:
Smart using these here. I like how the changes would cascade should we ever edit the button styling in the future. Nice.